styles/global.scss

```diff
@@ -111,7 +111,7 @@ body {
 }
 
 .w-100 {
-  width: 100%;
+  width: 100%!important;
```

Comment:
uff

Reply:
it was getting overriden in the case of buttons, but we want width 100% in that case